### PR TITLE
update BEP status

### PR DESCRIPTION
**Pull request by NathanBSC**

### CODE REVIEW

**Review by emailtovamos — APPROVED** [review]

File: BEPs/BEP227.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <pre>
     BEP: 227
     Title: Add BASEFEE opcode
-    Status: Pre-Draft
+    Status: Candidate
```
Comment (zzzckck) [inline BEPs/BEP227.md:4]: Enabled?

File: BEPs/BEP333.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <pre>
   BEP: 333
   Title: BNB Chain Fusion
-  Status: Draft
+  Status: Candidate
```
Comment (zzzckck) [inline BEPs/BEP333.md:4]: Enabled?

File: BEPs/BEP-299.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <pre>
   BEP: 299
   Title: Token Migration after BC Fusion
-  Status: Draft
+  Status: Candidate
```
Comment (zzzckck) [inline BEPs/BEP-299.md:4]: Enabled?

File: BEPs/BEP227.md (hunk 1 above)
Comment (NathanBSC) [reply BEPs/BEP227.md]: accept

**Review by zzzckck — APPROVED** [review]

**Pull request merged**